Cargo.lock

```diff
@@ -7254,18 +7271,6 @@ dependencies = [
  "windows-sys 0.59.0",
 ]
 
-[[package]]
-name = "wasmer-cache"
```

Comment:
Is this expected?

Reply:
Yes, we use our own FileSystemCache instead of it. see https://github.com/swc-project/swc/pull/10943/files#diff-3bda5def6ce2b7553c3b3a5ad241c0bdb7021e67b7de1e594df4cd5a54d403b3R248